study.md

```diff
@@ -29,21 +29,23 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop handles asynchronous code and queues it to run after all other
+code in the stack has cleared. When the stack is empty, the next item in the event loop will run, etc.
```

Comment:
Very close, the async code actually waits in the queue, the event loop watches for the stack to be empty and then moves the async bit from the queue to the stack.